pkg/controller/upload-controller.go

```diff
@@ -289,12 +291,18 @@ func (r *UploadReconciler) getOrCreateUploadPod(pvc *v1.PersistentVolumeClaim, p
 }
 
 func (r *UploadReconciler) getOrCreateScratchPvc(pvc *v1.PersistentVolumeClaim, pod *v1.Pod, name string) (*v1.PersistentVolumeClaim, error) {
+	// Set condition, then check if need to override with scratch pvc message
+	anno := pvc.Annotations
 	scratchPvc := &corev1.PersistentVolumeClaim{}
 	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: name, Namespace: pvc.Namespace}, scratchPvc); err != nil {
 		if !k8serrors.IsNotFound(err) {
 			return nil, errors.Wrap(err, "error getting scratch PVC")
 		}
 
+		anno[AnnRunningCondition] = "false"
+		anno[AnnLastTerminationMessage] = "Creating scratch space"
+		anno[AnnLastTerminationReason] = creatingScratch
+
```

Comment:
Not sure this ever makes sense.  Uploads should always have a scratch PVC and clones should never, right?